src/main/java/core/model/enums/RatingsStatistics.java

```diff
@@ -21,7 +16,7 @@
     public int getValue() {
         return value;
     }
-
+/*
```

Comment:
Just remove if this isn't needed any more.

Reply:
removed it